Grbl_Esp32/ProcessSettings.cpp

```diff
@@ -269,20 +265,16 @@ err_t report_startup_lines(const char* value, auth_t auth_level, ESPResponseStre
 }
 
 std::map<const char*, uint8_t, cmp_str> restoreCommands = {
-    #ifdef ENABLE_RESTORE_EEPROM_DEFAULT_SETTINGS
-        { "$", SETTINGS_RESTORE_DEFAULTS },
-        { "settings", SETTINGS_RESTORE_DEFAULTS },
-    #endif
-    #ifdef ENABLE_RESTORE_EEPROM_CLEAR_PARAMETERS
-        { "#", SETTINGS_RESTORE_PARAMETERS },
-        { "gcode", SETTINGS_RESTORE_PARAMETERS },
-    #endif
-    #ifdef ENABLE_RESTORE_EEPROM_WIPE_ALL
-        { "*", SETTINGS_RESTORE_ALL },
-        { "all", SETTINGS_RESTORE_ALL },
-    #endif
-    { "@", SETTINGS_RESTORE_WIFI_SETTINGS },
-    { "wifi", SETTINGS_RESTORE_WIFI_SETTINGS },
+#ifdef ENABLE_RESTORE_EEPROM_DEFAULT_SETTINGS
```

Comment:
I wonder why it removed the newlines here.  The tabular form seems like a help to readability.